ivadomed/loader/loader.py

```diff
@@ -267,8 +267,13 @@ def get_pair_metadata(self, slice_index=0, coord=None):
                     "data_type": 'gt',
                     "crop_params": {}
                 }))
-            elif len(gt_meta_dict):
-                gt_meta_dict.append(gt_meta_dict[0])
+            else:
+                gt_meta_dict.append(None)
```

Comment:
Could we add a comment here to explain why we are doing this trick? :-)

Reply:
Added here :-) b840602